swarm/src/protocols_handler.rs

```diff
@@ -101,11 +101,11 @@ pub use select::{IntoProtocolsHandlerSelect, ProtocolsHandlerSelect};
 /// continue reading data until the remote closes its side of the connection.
 pub trait ProtocolsHandler: Send + 'static {
     /// Custom event that can be received from the outside.
-    type InEvent: Send + 'static;
+    type InEvent: Debug + Send + 'static;
```

Comment:
If you would reference these as `fmt::Debug`, the imports would be slightly cleaner.

Reply:
I am fine either way. Applied suggestion in https://github.com/libp2p/rust-libp2p/pull/2183/commits/457635698abf9c05aa8fec069e8b49e4e5cad8db.